src/rules/property-no-unknown/README.md

```diff
@@ -85,3 +87,20 @@ The following patterns are *not* considered warnings:
   custom: 10px;
 }
 ```
+### `checkPrefixed: true | false` (default: `false`)
+
+This option forces to check vendor prefixed properties.
```

Comment:
My impression is that the justification is this: Many vendor-prefixed properties correspond to standard properties, but many others do not. In order to avoid maintaining a list of all existent vendor-prefixed properties, we are ignoring them altogether.